packages/fxa-auth-server/test/local/metrics/amplitude.js

```diff
@@ -225,12 +231,16 @@ describe('metrics/amplitude', () => {
           event: expectedEvent,
           context: expectedContext,
         });
+        sinon.assert.calledTwice(incrementSpy);
```

Comment:
There's a mixed approach to referencing the increment spy in this file. This one looks like the odd one out.

Reply:
That's true, a bunch of the others are using `assert.isTrue` with sinon, which will tell you nothing of utility if it fails except it was false. Using the sinon one gets the chai assert behavior and if it fails, it tells you useful information like the count difference, or what arg it was called with that has a mismatch. They should probably all be changed to using the more useful assert style with sinon, but that's a bit outside the scope of this PR.

How to use chai+sinon effectively to get useful feedback when asserts fail should probably be in a style guide for our code-base. Below on line 317 is an `assert.ok` on a boolean expression. If it fails, we have nothing of use. Instead it should use something like chai's `assert.isAbove` to verify one value is above another. It would then indicate the values in a failure.

Reply:
I agree... but I was just referring to `incrementSpy` vs `statsd.increment`

Reply:
Oh, in this case the container set was happening in the same function, that's why I did it this way. In the other tests the container is setup in a `beforeEach`, so it has to be retrieved by getting it again.

Reply:
I can make it look consistent though. :)